grails-app/views/base/_topNav.gsp

```diff
@@ -25,7 +25,7 @@
                     <li class="dropdown">
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown">New <b class="caret"></b></a>
                         <ul class="dropdown-menu">
-                            <li><a href="${createLink(controller: 'project', action: 'pages')}">New Project</a></li>
+                            <li><a href="${createLink(controller: 'project', action: 'newProject')}">New Project</a></li>
```

Comment:
this change is no longer related to this story. Do we really need it ?

Reply:
No we don't for this story, I've reverted all changes related to moving NewConsentGroup to React Routes.